SparkPost/2web2ui
# SA-1584: Saved Reports list

### What Changed
 - Add list modal for reports
 - Links for modal close modal and set new report options
 - Report options

### How To Test
 - Go to report builder
 - Enable UI option `allow_saved_reports`
- Click on View All Reports
 - Verify modal pops up
 - Verify tabs swap between tables
 - Verify that links work correctly

### To Do
- [ ] Matchbox adds padding to modal content. Maybe swap out to old Modal API?


## Notes
 Matchbox has 500 padding to the modal content and it can't be modified in current matchbox API. Might be worth pinging UX over.
